01_Core/week02/submissions/index.html

```diff
@@ -0,0 +1,109 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>Cassidy's Webpage</title>
+        <link rel="stylesheet" href="style.css">
+    </head>
+    <div class="main">
+    <header id="home">
+        <div class="nav-bar">
+            <a href="#home" class="btn">Home</a>
+            <a href="#about" class="btn">About</a>
+            <a href="#career" class="btn">Career</a>
+            <a href="#you" class="btn">About You</a>
+            <div class="dropdown">
+                <button class="dropbtn">Social Media</button>
+                    <div class="dropdown-content">
+                        <a href="#">LinkedIn</a>
+                        <a href="#">Twitter</a>
+                        <a href="#">Facebook</a>
+                    </div>
+            </div>
+        </div>
+        <div class="sidenav">
```

Comment:
Is there a semantic HTML tag that indicates navigation? Maybe this should also be nested inside of a semantic HTML tag indicating that it is an aside to the main content?